packages/core/content-manager/server/services/entity-manager.js

```diff
@@ -171,21 +171,27 @@ module.exports = ({ strapi }) => ({
       publishData[PUBLISHED_AT_ATTRIBUTE] = null;
     }
 
-    const params = { data: publishData, populate: getDeepPopulate(uid) };
+    const params = {
+      data: publishData,
+      populate: getDeepPopulate(uid, null, { onlyMany: true, countMany: true }),
```

Comment:
Do we only need to populate `xToMany` relationships in CM?
Are the `xToOne` relationships not displayed?

Reply:
Yes :)
Relations are not populated anymore, only the count for `xToMany` is useful for the front.
The `xToOne` relations could be populated but since the front fetches them anyway it is not useful for them to populate it.